controllers/acesFile/acesFile.go

```diff
@@ -0,0 +1,45 @@
+package acesFile
+
+import (
+	"bytes"
+	"strconv"
+	"time"
+
+	"github.com/curt-labs/API/helpers/encoding"
+	"github.com/curt-labs/API/helpers/error"
+	"github.com/curt-labs/API/models/acesFile"
+	"github.com/curt-labs/API/models/brand"
+	"github.com/go-martini/martini"
+
+	"net/http"
+)
+
+func GetAcesFile(rw http.ResponseWriter, req *http.Request, enc encoding.Encoder, params martini.Params) string {
+	version := params["version"]
+
+	var brandObj brand.Brand
+	brandID, err := strconv.Atoi(req.URL.Query().Get("brandID"))
```

Comment:
The middleware already gets the brandID and sets it in a datacontext struct that you can inject into your handler.

https://github.com/curt-labs/API/blob/goapi/controllers/middleware/middleware.go#L153

example usage: https://github.com/curt-labs/API/blob/goapi/controllers/webProperty/webProperty_controller.go#L95